packages/phone-number-privacy/combiner/package.json

```diff
@@ -28,12 +28,20 @@
     "test:e2e:alfajores": "CONTEXT_NAME=alfajores yarn test:e2e"
   },
   "dependencies": {
-    "@celo/contractkit": "^4.1.1-dev",
```

Comment:
you might need to revert some of these changes in order to merge to master

Reply:
Reverted, thanx!